proto/spacemesh/smesher_types.proto

```diff
@@ -1,6 +1,6 @@
 syntax = "proto3";
 package spacemesh;
-
+import "spacemesh/types.proto";
```

Comment:
Doesn't seem required, is it?

Reply:
hm.. It is removed in my branch - maybe I need to push a bit more changes.